index.js

```diff
@@ -0,0 +1,143 @@
+cardValue = {
+    KING: 10,
+    QUEEN: 10,
+    JACK: 10,
+    ACE: [1, 11],
+    "1": 1,
+    "2": 2,
+    "3": 3,
+    "4": 4,
+    "5": 5,
+    "6": 6,
+    "7": 7,
+    "8": 8,
+    "9": 9,
+    "10": 10
+}
+
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
"DOMContentLoaded" is an event, and when it happens a function should be executed . Your function is empty in this case. Your code still runs, because there's not that much of it. But when you'll be building bigger projects, you need to be placing all the functionality of your js file inside the function (starting line 19 in this case).